### PR TITLE
Add Project Management

### PR DESCRIPTION
**Pull request by dend**
Adding a list for anyone to learn more about product management.

**https://github.com/dend/awesome-product-management**

**The list presents resources for anyone to start with product management. It contains pointers to good articles, tools, books and podcasts that can help accelerate one's PM training.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -436,6 +436,7 @@
 - [Educational Games](https://github.com/yrgo/awesome-eg#readme) - Learn while playing.
 - [JavaScript Learning](https://github.com/micromata/awesome-javascript-learning#readme)
 - [CSS Learning](https://github.com/micromata/awesome-css-learning#readme) - Mainly about CSS – the language and the modules.
+- [Product Management](https://github.com/dend/awesome-product-management) - learn about resources to be a great product manager.
```
Comment (steren) [inline readme.md]: I would rephrase to something more descriptive: Resources for Product Manager: tools, articles, books

**Comment by dend** [comment]
@brentru - likely an issue with the linter. If you open the link, you'll notice that it works correctly.

File: readme.md (hunk 1 above)
Comment (dend) [reply readme.md]: Updated, good feedback!

**Comment by sindresorhus** [comment]
> You have to review at least 2 other open pull requests.

---

> This pull request has a descriptive title in the format Add Name of List (Example: Add Swift), not Update readme.md or Add awesome list.

File: readme.md [hunk 2]
```diff
@@ -436,6 +436,7 @@
 - [Educational Games](https://github.com/yrgo/awesome-eg#readme) - Learn while playing.
 - [JavaScript Learning](https://github.com/micromata/awesome-javascript-learning#readme)
 - [CSS Learning](https://github.com/micromata/awesome-css-learning#readme) - Mainly about CSS – the language and the modules.
+- [Product Management](https://github.com/dend/awesome-product-management) - Resources for Product Managers: tools, articles, books, podcasts and more.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
Include a short description about the project/theme of the list. **It should not describe the list itself.**

Comment (sindresorhus) [inline readme.md]:
```suggestion
- [Product Management](https://github.com/dend/awesome-product-management#readme) - Resources for Product Managers: tools, articles, books, podcasts and more.
```

Comment (dend) [reply readme.md]: @sindresorhus - addressed this, thank you for the feedback!

Comment (dend) [reply readme.md]: @sindresorhus addressed this as well - thanks again for looking over the PR!

**Comment by dend** [comment]
@sindresorhus the 2 reviews requirement should now be met.

**Comment by sindresorhus** [comment]
The heading title of your awesome list should be `Awesome Product Management`.

---

It's not clear how the Sketch app is related to project management.

---

`Url` => `URL`

---

Don't use my email in https://github.com/dend/awesome-product-management/blob/master/code-of-conduct.md

**Comment by dend** [comment]
@sindresorhus latest changes to the repo should address your concerns. Regarding the Sketch inclusion in the list - on our team, product managers use it to prototype quickly and create artifacts that can demonstrate product vision and ideas around it.

**Comment by sindresorhus** [comment]
There should be a description at the top of your list what a product/program manager is.

---

I think the items in the Podcasts section could use a description. Also items in the Conferences section.

**Comment by dend** [comment]
@sindresorhus - updated the `README.md` with the requested changes, thanks for reviewing the doc!

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1132529556412178432

**Comment by dend** [comment]
Amazing - thank you @sindresorhus! 🎉
